platform_design/lib/songs_tab.dart

```diff
@@ -138,7 +139,7 @@ class _SongsTabState extends State<SongsTab> {
         CupertinoSliverNavigationBar(
           trailing: CupertinoButton(
             padding: EdgeInsets.zero,
-            child: Icon(CupertinoIcons.shuffle),
+            child: Icon(CupertinoIcons.shuffle_thick),
```

Comment:
Can we add a todo here to change this back when ready?

Reply:
Sure thing.